pyrakoon/__version__.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) iNuron - info@openvstorage.com
+# This file is part of Open vStorage. For license information, see <LICENSE.txt>
+
+from .__hash__ import __hash__
+
+# Used by setup.py to compute the version
+__version_info__ = 0, 0, 1
+# Version with the added hash
+__version_full__ = __version_info__ + (__hash__,)
+__version_tuple__ = tuple(str(i) for i in __version_full__)
```

Comment:
why convert to string ? (semver states they should be nonnegative integers, except for the 4th component)